LoRaEngine/modules/LoRaWanPktFwdModule/module.json

```diff
@@ -4,7 +4,7 @@
   "image": {
     "repository": "$CONTAINER_REGISTRY_ADDRESS/lorawanpktfwdmodule",
     "tag": {
-      "version": "0.1.0-preview",
+      "version": "0.1.1-preview",
```

Comment:
Should we switch "preview" to "dev" like 0.2.1-dev to harmonize with GitHub and avoid errors in case we patch preview branch containers.

Reply:
You are right, I changed it to 0.2.0-preview